tabbycat/standings/tests/test_standings.py

```diff
@@ -253,11 +252,9 @@ def setUp(self):
         debate = self.set_up_ignorable_debate()
         debate.ballotsubmission_set.update(confirmed=False)
 
-    @expectedFailure
     def test_draw_strength(self):
         super().test_draw_strength()
```

Comment:
So if we overrode this "properly", it would be:
```suggestion
    def test_draw_strength(self):
        # The ignorable debate still counts as an opponent (because they definitely faced each
        # other) but does not count as a win (because it's unconfirmed). So the losing team has
        # faced the winning team 3 times (including ignorable debate), and the winning team has 2
        # wins (excluding ignorable debate), for a losing team draw strength of 3 * 2 = 6.
        self._base_metric_test({'draw_strength': [0, 6]})
```